back/engines/commercial/user_custom_fields/app/models/user_custom_fields/representativeness/r_score.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+class UserCustomFields::Representativeness::RScore
+  attr_reader :value, :user_counts, :ref_distribution
+
+  # @param [Numeric] value
+  # @param [UserCustomFields::Representativeness::RefDistribution] ref_distribution
+  def initialize(value, user_counts, ref_distribution)
+    @value = value
+    @user_counts = user_counts
+    @ref_distribution = ref_distribution
+    @timestamp = Time.zone.now
+  end
+
+  def id
+    "#{@ref_distribution.id}_#{@timestamp.to_i}_rscore"
+  end
+
+  class << self
+    # Factory method that instantiates an RScore based on observed user counts and a reference distribution.
+    # @param [Hash] user_counts
+    # @param [UserCustomFields::Representativeness::RefDistribution] ref_distribution
+    # @return [UserCustomFields::Representativeness::RScore]
+    def compute(user_counts, ref_distribution)
+      population_counts = ref_distribution.distribution
+      score_value = compute_scores(user_counts, population_counts)[:min_max_p_ratio]
```

Comment:
I thought we were going for the `min_mean_p_ratio`?

Reply:
Yes think stick to min/max for now